README.adoc

```diff
@@ -70,156 +60,77 @@ This guide is a work in progress - existing guidelines are constantly being impr
 guidelines are being added, and occasionally some guidelines would get removed.
 
 
-== Why Packaging Extension?
```

Comment:
I think the rationale makes sense, but you should simply remove the references to RuboCop here.

Reply:
Ah, fair enough, done! 😄